internal/api/schemas/schemas.go

```diff
@@ -269,10 +269,20 @@ func (s *sch) validatePlugin(plugin common.Plugin, modelKind string, modelName s
 }
 
 func (s *sch) init() error {
-	for _, l := range s.loaders {
-		if err := l.Load(); err != nil {
+	return RunLoaders(s.loaders, "model", "full")
```

Comment:
it doesn't seem there is something that prevents you to use the constant you defined in the file `monitoring` :)